packages/@aws-cdk/aws-applicationautoscaling/test/scalable-target.test.ts

```diff
@@ -79,6 +79,35 @@ describe('scalable target', () => {
     });
   });
 
+  test('add scheduled scaling with timezone support', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const target = createScalableTarget(stack);
+
+    // WHEN
+    target.scaleOnSchedule('ScaleUp', {
+      schedule: appscaling.Schedule.rate(cdk.Duration.minutes(1)),
+      maxCapacity: 50,
+      minCapacity: 1,
+      timezone: 'America/New_York',
```

Comment:
what happens when you specify an invalid timezone? I don't think its a good user experience to have a deploy-time failure.

Reply:
@kaizencc  do you think we should create a enum-like class that includes all timezones defined in the [tz database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones)?

Reply:
I guess description is all we need.